### ISSUE TITLE
[Bug - next-pwa]: __PWA_SW_ENTRY_WORKER__ undefined for the new 10.2.1 build with pwa disable

### ISSUE THREAD

**Issue by roderickhsiao** [DuCanhGH/next-pwa#136]
### Provide environment information

Thanks for building the package, we notice an exception is thrown

```
Uncaught ReferenceError: __PWA_SW_ENTRY_WORKER__ is not defined
    at eval (sw-entry.js:3:95)
    at ./node_modules/@ducanh2912/next-pwa/dist/sw-entry.js (main.js?ts=1704998833829:1263:1)
    at options.factory (webpack.js?ts=1704998833829:736:31)
    at __webpack_require__ (webpack.js?ts=1704998833829:37:33)
    at __webpack_exec__ (main.js?ts=1704998833829:1643:61)
    at main.js?ts=1704998833829:1644:37
    at webpackJsonpCallback (webpack.js?ts=1704998833829:1408:39)
    at main.js?ts=1704998833829:9:61
```

our PWA config 

```
const withPWA = require('@ducanh2912/next-pwa').default({
  aggressiveFrontEndNavCaching: true,
  cacheOnFrontendNav: true,
  cacheStartUrl: false,
  dest: 'public',
  disable: process.env.NODE_ENV === 'development',
});
```

### Link to reproduction - Issues with a link to complete (but minimal) reproduction code help us address them faster

https://github.com/roderickhsiao/next-pwa-debug

### To reproduce

We are on Next 14.0.4 using page folder.

It seems like with `disable` option, the code logic is partially executed instead of skipped completely. 
downgrading the package to 10.1.0 works fine.

### Describe the bug

Javascript exception 

### Expected behavior

When `disable` option is true, the PWA service worker logic shouldn't kicks in and client wont load sw logic

**Comment by DuCanhGH** [comment]
Should be fixed in 10.2.2 :)

**Comment by roderickhsiao** [comment]
Verified! thanks for the quick fix
